docs/INSTALLATION.md

```diff
@@ -7,7 +7,7 @@ it from [the official site](https://nodejs.org/). Once you have Node.js up and
 running, you can install elm and elm-test using 
 
 ```bash
-npm install -g elm elm-test
+npm install -g elm elm-test@beta
```

Comment:
Actually, since we are using elm 0.19, we should be more specific and use `elm0.19.0` There are 4 different tags you can see [here](https://www.npmjs.com/package/elm-test/v/0.19.0-beta11). This will help us later when upgrading to the next version.

Reply:
:+1: